packages/common/src/index.ts

```diff
@@ -12,6 +12,7 @@ export * from './converter/data-converter';
 export * from './converter/failure-converter';
 export * from './converter/payload-codec';
 export * from './converter/payload-converter';
+export * from './converter/payload-search-attributes';
```

Comment:
Let's be specific in what we reexport. TBH, I don't think there's anything in that file that I would consider as public API. It's ok for our own packages to import inner files of the common package.

Reply:
Yeah I agree.

Haven't commit this change yet, getting some weird import issues after replacing with `@temporalio/common/lib/converter/payload-search-attributes`, but I'll add it once I fix it.

Reply:
Done